<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogExportParams</class>
 <widget class="QDialog" name="DialogExportParams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Specify video parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupVideo">
         <property name="title">
          <string>Video encoding parameters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="3">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Quality</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Target medium</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="boxVideoProfile">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="boxVideoMedium">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QComboBox" name="boxVideoTarget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QComboBox" name="boxVideoQuality">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Subtarget</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Audio</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Video profile</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QComboBox" name="boxVideoAudio">
            <item>
             <property name="text">
              <string>encode according to profile</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>copy original content</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>no audio</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="3" colspan="2">
           <widget class="QLabel" name="lblVideoDetailsLink">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;yes&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#006e28;&quot;&gt;Click here to see the profile details&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupCDG_2">
         <property name="title">
          <string>Lyrics rendering</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>Font:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFontComboBox" name="fontVideo">
              <property name="whatsThis">
               <string>Font family for preview window</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>Size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="fontVideoSize">
              <property name="whatsThis">
               <string>Font size for preview window. Larger values are suggested (24 and up)</string>
              </property>
              <property name="minimum">
               <number>4</number>
              </property>
              <property name="maximum">
               <number>120</number>
              </property>
              <property name="singleStep">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnDetectSize">
              <property name="text">
               <string>Detect size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnTestSize">
              <property name="text">
               <string>Test size</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>Keep title at least </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="titleVideoMin">
              <property name="maximum">
               <number>120</number>
              </property>
              <property name="value">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>seconds</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>Colors</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="1" column="2">
              <widget class="QLabel" name="label_31">
               <property name="text">
                <string>Not sung yet</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="ColorButton" name="btnVideoColorInfo">
               <property name="whatsThis">
                <string>Preview window text color for the text which has been sung.</string>
               </property>
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="ColorButton" name="btnVideoColorActive">
               <property name="whatsThis">
                <string>Preview window text color for the text which has been sung.</string>
               </property>
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_30">
               <property name="text">
                <string>Sung</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="ColorButton" name="btnVideoColorInactive">
               <property name="whatsThis">
                <string>Preview window text color for the text which is not being sung yet..</string>
               </property>
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_29">
               <property name="text">
                <string>Information</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="ColorButton" name="btnVideoColorBg">
               <property name="whatsThis">
                <string>Preview window background color.</string>
               </property>
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_28">
               <property name="text">
                <string>Backround</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="cbVideoPreamble">
              <property name="text">
               <string>Draw squares showing where singing starts</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="boxEnableAntialiasing">
              <property name="text">
               <string>Enable anti-aliasing</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Title window parameters</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Artist</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leArtist"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leTitle"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Created by:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leTitleCreatedBy"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Output file</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblOutput">
            <property name="text">
             <string>Write video to file:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leOutputFile"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnBrowse">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Preview</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lblImage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>5</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lblCurrent">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="seekSlider">
           <property name="maximum">
            <number>199</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblTotal">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>boxVideoMedium</tabstop>
  <tabstop>boxVideoTarget</tabstop>
  <tabstop>boxVideoProfile</tabstop>
  <tabstop>boxVideoQuality</tabstop>
  <tabstop>boxVideoAudio</tabstop>
  <tabstop>fontVideo</tabstop>
  <tabstop>fontVideoSize</tabstop>
  <tabstop>btnDetectSize</tabstop>
  <tabstop>btnTestSize</tabstop>
  <tabstop>titleVideoMin</tabstop>
  <tabstop>btnVideoColorBg</tabstop>
  <tabstop>btnVideoColorInactive</tabstop>
  <tabstop>btnVideoColorInfo</tabstop>
  <tabstop>btnVideoColorActive</tabstop>
  <tabstop>leArtist</tabstop>
  <tabstop>leTitle</tabstop>
  <tabstop>leTitleCreatedBy</tabstop>
  <tabstop>leOutputFile</tabstop>
  <tabstop>btnBrowse</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>seekSlider</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogExportParams</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogExportParams</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
